system_setup/client/controllers/identity.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2015-2021 Canonical, Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+
+from subiquity.client.controllers import IdentityController
+from subiquity.common.types import IdentityData
+from system_setup.ui.views import WSLIdentityView
+
+log = logging.getLogger('system_setup.client.controllers.identity')
+
+
+class WSLIdentityController(IdentityController):
```

Comment:
I don't think this gains anything from inheriting from subiquity's identity controller.

Reply:
Ack. WSLIdentityController is now directly inheriting from SubiquityTuiController.